src/Illuminate/Support/Env.php

```diff
@@ -174,6 +174,32 @@ public static function writeVariable(string $key, mixed $value, string $pathToFi
         $filesystem->put($pathToFile, implode(PHP_EOL, $lines));
     }
 
+    /**
+     * Remove a single key from the environment file.
+     *
+     * @param  string  $key
+     * @param  string  $pathToFile
+     * @return void
+     *
+     * @throws RuntimeException
+     * @throws FileNotFoundException
+     */
+    public static function removeVariable(string $key, string $pathToFile): void
+    {
+        $filesystem = new Filesystem;
+
+        if ($filesystem->missing($pathToFile)) {
+            throw new RuntimeException("The file [{$pathToFile}] does not exist.");
+        }
+
+        $envContent = $filesystem->get($pathToFile);
+
+        $lines = explode(PHP_EOL, $envContent);
+        $lines = array_filter($lines, fn ($line) => ! str_starts_with($line, $key.'='));
```

Comment:
This is incorrect. The key can have extra whitespace after, and the value could be multi-line.